APE9.rst

```diff
@@ -0,0 +1,111 @@
+A General Timeseries Class for Astropy
+-------------------------------------
+
+author: Stuart Mumford
+
+date-created: 2016 March 24
+
+date-last-revised: 2016 March 24
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+The goal of a timeseries object in astropy is to provide a core set of
+functionality that can be shared between astropy, affiliated packages and the
+wider community.
+
+The scope of this timeseries object is designed to remain focused on the core
+functionality needed, and not to try and replace or re-implement the
+functionality in the pandas library. The main motivation for this class over
+using pandas is primarily to support the functionality of other parts of the
+astropy package such as Time, Units and Coordinates packages.
+
+This document proposes an implementation based on Table, with extra constraints
+and specific functionality dedicated to timeseries type data.
+
+Detailed description
+--------------------
+
+Many different areas of astrophysics have to deal with 1D timeseries data, i.e.
+either sampling a continuous variable at fixed times or counting some events
+binned into time windows. These types of applications require some basic
+functionality:
```

Comment:
How many of these features already exist in `QTable`?

Reply:
Some, but all would need to be customised for timeseries data (i.e. to preserve the index correctly.)